app/views/posts/index.html.erb

```diff
@@ -1 +1,36 @@
-<h1>Posts</h1>
\ No newline at end of file
+<section>
+  <ul>
+    <%= render @user %>
+  </ul>
+</section>
+<section class="">
+  <ul class="posts_list">
+    <% @user.posts.each_with_index do |post, i| %>
+      <li class="single_post">
+        <div class="post-header">
+          <h3>Post #<span><%= i + 1 %> </span>
```

Comment:
- [ ] To make your code professional, Kindly remove commented code in this line :)

Reply:
Hi @Rustamxon7  this is not commented code. If you checked the view it displays

`Post # 1`